doc/cygnus-ngsi-ld/flume_extensions_catalogue/ngsi_postgis_sink.md

```diff
@@ -0,0 +1,169 @@
+# <a name="top"></a>NGSIPostGISSink
```

Comment:
This .md file should be added to mkdocs.yml so it can be rendered by RTD.

Reply:
Ok, I included it in the documentation